mk/config.mk

```diff
@@ -1284,3 +1284,14 @@ endif
 # handling of external abort implementing the plat_external_abort_handler()
 # function.
 CFG_EXTERNAL_ABORT_PLAT_HANDLER ?= n
+
+# CFG_CORE_LIBGCC, when enabled, links OP-TEE core and ldelf with libgcc.
+# Linking OP-TEE core and ldelf with libgcc is deprecated, but keep this
+# flag while sorting out remaining issues with removing the libgcc
+# dependency.
+CFG_CORE_LIBGCC ?= n
+
+# CFG_TA_LIBGCC, when enabled, links user mode TAs with libgcc. Linking
+# TAs with libgcc is deprecated, but keep this flag while sorting out the
+# out remaining issues with supporting C++.
```

Comment:
Yes! We're deprecating the libgcc support. This essentially means that we deprecate the C++ support as well if we're not able to come up with another solution for that. Besides concluding the challenges with C++, I don't think we have a great plan for it right now.

Perhaps we should ask ourselves how many OP-TEE users we believe are using C++. C is the primary language and for additional language support, it might be better to invest time in Rust rather than trying to "retrofit" C++. Thoughts?